utils/filters/job.filters.js

```diff
@@ -0,0 +1,24 @@
+export const isRemoteFilter = {
+  filterName: "isRemote",
+  filterFn: (item) => item.location.includes("remote"),
+};
+
+export const createIsMineFilter = (user) => ({
+  filterName: "isMine",
+  filterFn: (item) => item.user_id === user.id,
+});
+
+export const createIsNotMineFilter = (user) => ({
```

Comment:
As I'm thinking through some interactions with this, I think we can scrap it and just stick with the "isMine" toggle.

Reply:
good call, on it